packages/jupyter-ai/jupyter_ai/document_loaders/directory.py

```diff
@@ -125,17 +129,20 @@ def split(path, all_files: bool, splitter):
                     d for d in subdirs if not (d[0] == "." or d in EXCLUDE_DIRS)
                 ]
                 filenames = [f for f in filenames if not f[0] == "."]
-            filepaths += [Path(os.path.join(dir, filename)) for filename in filenames]
+            filepaths.extend([Path(dir) / filename for filename in filenames])
+    VALID_EXTS = {j.lower() for j in SUPPORTED_EXTS}
```

Comment:
Accordingly to PEP 8,
> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL. 

https://peps.python.org/pep-0008/#constants

Convention is to use all caps for module-level constants. So Using all caps for a variable defined inside a function even if it's meant to be a constant within the context of the function is a bit unconventional and threw me off / might throw other people off. Due clarity and consistency with convention, defining variable in a standard non-all-caps way for example `valid_extensions` would be better.

Reply:
@andrii-i Thanks for this guidance. I have made the change.